keyboards/rmkeebs/rm_numpad/readme.md

```diff
@@ -0,0 +1,29 @@
+# rm_numpad
+
+<!---
+TODO: update image when final product is ready for sale.
+-->
+![rm_numpad](https://i.imgur.com/1hqUHIn.jpg)
```

Comment:
```suggestion
![rm_numpad](https://i.imgur.com/1hqUHInh.jpg)
```

Reply:
I'm pushing a commit with a link to a different image.  I've sized it 1024px wide, like you did here.

Reply:
https://github.com/qmk/qmk_firmware/pull/15983/commits/72e4c1acfac09f15516d89f572dfc0f6bce7e4ee